app/Models/Ticket.php

```diff
@@ -31,6 +31,7 @@ class Ticket extends BaseModel
     // TODO rename Updated column to updated_at
     // TODO drop AchievementID, use ticketable morph instead
     // TODO drop Hardcore, derived from player_session
+    // TODO rename ticketable_model to ticketable_type
```

Comment:
`ticketable_model` implies we'll be storing a fully-qualified model name to the database, ie: `"App\\Models\\Achievement"`.

I think `ticketable_type` is a more desirable field, as it can just be an enum.